nym-wallet/README.md

```
@@ -41,9 +41,10 @@ If problems persist on Wayland, try preloading the system client library (path m
 
 ## Installation
 
-Inside the `nym-wallet` directory, run the following command:
+From the `nym` repo root (pnpm workspace), run:
+
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)